pages/common/git-show-ref.md

```diff
@@ -0,0 +1,20 @@
+# git show-ref
+
+> Git command for listing references.
+> More information: <https://git-scm.com/docs/git-show-ref>.
+
+- Show all refs in the repository:
+
+`git show-ref`
+
+- Show only heads references:
```

Comment:
Perhaps this will read better?
```suggestion
- Show only references with an associated head:
```

Reply:
I don't think so, because of two reasons:
1- This is the language used by git itself in different git pages, you can see for example here https://git-scm.herokuapp.com/book/zh-tw/v2/Git-Internals-Git-References that they use ```head references```.
2- a ```ref with an associated head``` might give the wrong impression that a ref might be associated with more than just a head or that it can be empty, but both of these can not happen.

Reply:
Ah, I see. Let's leave it as it is then.